From 0be2ac89c4ab4e9c319dc365710e9a19b1a00bd0 Mon Sep 17 00:00:00 2001
From: Zhao Wei Liew <zhaoweiliew@gmail.com>
Date: Sun, 26 Jun 2016 11:45:11 +0800
Subject: [PATCH 1/3] power: Don't send obsolete DISPLAY_OFF opcode

This opcode is obsolete as perflock doesn't actually handle it.
It was originally meant for 8974, but was replaced later on.

The exact same opcode was removed for the interactive governor
a while back, so remove it for the ondemand governor as well.

Change-Id: I7bc5d6f28a827d3e8c83acde1319577db3d06ea6
---
 power-helper.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/power-helper.c b/power-helper.c
index da544de..3130012 100644
--- a/power-helper.c
+++ b/power-helper.c
@@ -375,7 +375,7 @@ void power_set_interactive(int on)
         /* Display off. */
         if ((strncmp(governor, ONDEMAND_GOVERNOR, strlen(ONDEMAND_GOVERNOR)) == 0) &&
                 (strlen(governor) == strlen(ONDEMAND_GOVERNOR))) {
-            int resource_values[] = {DISPLAY_OFF, MS_500, THREAD_MIGRATION_SYNC_OFF};
+            int resource_values[] = { MS_500, THREAD_MIGRATION_SYNC_OFF };
 
             if (!display_hint_sent) {
                 perform_hint_action(DISPLAY_STATE_HINT_ID,
-- 
2.15.1

